docs/api/tutorials/sdk/search_client.md

```diff
@@ -0,0 +1,157 @@
+# Search
+
+DataHub’s Python SDK makes it easy to search and discover metadata across your data ecosystem. Whether you're exploring unknown datasets, filtering by environment, or building advanced search tools, this guide walks you through how to do it all programmatically.
+
+## What You Can Do
+
+- Search metadata by keyword or structured filters
+- Filter by environment, platform, type, or custom properties
+- Use AND / OR / NOT logic for advanced queries
+- Perform targeted searches using custom fields like URN
+
+## Getting Started
+
+Install the SDK:
+
+```bash
+pip install acryl-datahub
+```
+
+Then, connect to your DataHub instance:
+
+```python
+from datahub.sdk import DataHubClient
+
+client = DataHubClient(server="<your_server>", token="<your_token>")
+```
+
+If authentication is enabled, you’ll need to generate a Personal Access Token from your DataHub instance. [See the guide](https://docs.datahub.com/docs/authentication/personal-access-tokens).
```

Comment:
also - in general, I don't like links where the text is "see here" or similar. the anchor text should give some indication of where the link goes

```suggestion
If authentication is enabled, you’ll need to [generate a Personal Access Token](https://docs.datahub.com/docs/authentication/personal-access-tokens) from your DataHub instance.
```